Projects/TEAM-3/Turbal/tasks_progects/task5/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿//5. Введіть три числа і виведіть на екран значення суми і добуток цих чисел.
+
+float n1 = 0, n2 = 0, n3 = 0, sum = 0, mult = 0;
+bool is_correct = true;
+
+Console.WriteLine("Please, input first number:");
+try
+{
+    n1 = Convert.ToSingle(Console.ReadLine());
+}
+catch
+{
+    Console.WriteLine("First input is not a number!");
+    is_correct = false;
+}
+
+Console.WriteLine("Please, input second number:");
+try
+{
+    n2 = Convert.ToSingle(Console.ReadLine());
+}
+catch
+{
+    Console.WriteLine("Second input is not a number!");
+    is_correct = false;
+}
+
+Console.WriteLine("Please, input third number:");
+try
+{
+    n3 = Convert.ToSingle(Console.ReadLine());
+}
+catch
+{
+    Console.WriteLine("Second input is not a number!");
+    is_correct = false;
+}
+
+if (is_correct == true)
```

Comment:
you can use it like this 
if (is_correct)

Reply:
ok, thanks!